src/App.js

```diff
@@ -1,24 +1,26 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { useState } from "react";
+
+//COMPONENTS
+import ReviewList from "./components/ReviewList";
+import NavBar from "./components/NavBar";
+
+//USER CONTENT
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <h1>Welcome to HB- Gaming Reviews</h1>
       </header>
-    </div>
+      <NavBar />
+      <p>Take a look at our lastest reviews</p>
+      <ReviewList />
```

Comment:
You may want to add this component inside a Route and give it the path of "/" (if you want all your reviews to load on the "home page").

Otherwise when you start to move onto other stories you will always render this component at the moment on every page!